rails/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+  rails:
+    build: ./
+    image: ${DOCKER_IMAGE_NAME-kaiju-rails}
+    environment:
+      - SECRET_KEY_BASE=980453e07f6f1a473091a79dc0c5109d6e350509a5b47d96b9bc55ce40e3b45ce2959919e9e528602c3a7a7c169c5e01596feeb8328ef51063514641b964d917
+      - REDIS_URL=redis://redis:6379/0
+      - RAILS_SERVE_STATIC_FILES=true
+      - KAIJU_ALLOW_NO_AUTH=i_accept_the_risk_of_running_with_no_authentication
+    depends_on:
+      - redis
+    networks:
+      - kaiju
+    ports:
+      - 3000:3000
+
+  redis:
+    image: redis:3.2
+    networks:
+      - kaiju
+    command: redis-server
+
+  test:
+    build: ./
+    image: ${DOCKER_IMAGE_NAME-kaiju-rails}
```

Comment:
Is there a default fallback that could be added so this can be run locally as well?

Reply:
The -kaiju-rails is the fallback for running locally, it works as an OR operator and will build an image named kaiju-rails if the docker image name is not provided

Reply:
When you add both `build` and `image` in the same service declaration `image` is the name the `build` is tagged with.